cinema/models.py

```diff
@@ -35,12 +39,21 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_file_path(instance, filename):
+    _, extension = os.path.splitext(filename)
+
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}.{extension}"
```

Comment:
Is dot really needed here? Check files)

Reply:
media/uploads/movies/the-departed-720310f8-e981-4201-8860-af4d4512b58e..jpg
Example